README.md

```diff
@@ -76,3 +77,67 @@ issues, check the logs for the `gcp-cost-control` Cloud Function.
     "currencyCode": "USD"
 }
 ```
+
+## Daily cost reports
+
+The [gcp_cost_report](gcp_cost_report.py) Cloud Function can be used to get a
+daily per-project cost report in Slack.
+
+1. Set up [Cloud Billing data export to BigQuery]
+   (https://cloud.google.com/billing/docs/how-to/export-data-bigquery) in the
```

Comment:
Probably redundant spaces here, break the Markdown link formatting :)

Reply:
Nice catch!